docker/function-image-builder/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM fedora:27
```

Comment:
Is there a reason, why to use fedora? Due to corp requirements, we probably have to build our own images, and dealing with different base images will get tricky.

Reply:
The only reason is because it is the only platform that has `skopeo` as a system package. In case we want to use a different distribution we need to build it from source and include the same dependencies in the other image.

Reply:
ok, thank you. We might do that at a later point, if we are going through security audit.